scripts/peering-prefix

```diff
@@ -26,6 +26,20 @@ check_valid_asn () {  # {{{
     fi
 }  # }}}
 
+change_client_nexthop () {
+    local mux=$1
+    local mux2dev
+    mux2dev=$(grep "$mux" "$mux2dev_db")
```

Comment:
Please use `load_mux2dev` (inside `peering-config`) to load the associative array, and then use something like:

```{bash}
        dev=${mux2dev[$mux]}
        devid=${dev##tap}
```